tests/test_organization.py

```diff
@@ -179,3 +179,47 @@ def test_create_relationship(setUp,
 
     fake_signal.assert_called_with(object_id=composition.id,
                                    sender=Composition)
+
+def is_tab_active(page, tab_name):
+    if 'primary">{}'.format(tab_name) in page.content.decode('utf-8'):
+        return True
+    else:
+        return False
+
+
+@pytest.mark.django_db
+def test_organization_edit_buttons(setUp,
+                                   full_organizations,
+                                   membership_person,
+                                   membership_organization):
+     org = full_organizations[0]
```

Comment:
Quick note that my linter is warning that the tabs from here on down are five spaces instead of four.

Reply:
hm, it looks like my entire file is indented with five spaces. that's not the case for you?

Reply:
I only see five spaces for this test, as you can see comparing it to the code blocks above it:

<img width="1042" alt="Screen Shot 2019-08-19 at 12 15 50 PM" src="https://user-images.githubusercontent.com/14170650/63285350-1f18dd80-c27b-11e9-8548-40ae3940c73b.png">

Might be something to do with your editor configuration?

Reply:
i made some changes in my editor and now i'm seeing four spaces for the entire file—i committed that!